text/src/autogluon/text/automm/utils.py

```diff
@@ -820,9 +821,7 @@ def load_text_tokenizers(
             per_path = os.path.join(path, per_text_processor.tokenizer)
             per_text_processor.tokenizer = per_text_processor.get_pretrained_tokenizer(
                 tokenizer_name=per_text_processor.tokenizer_name,
-                checkpoint_name=per_path,
-            )
-
+                checkpoint_name=per_path)
```

Comment:
This change is unnecessary and violates the format of `()` in our code.

Reply:
Just say that we may fix all the style issues in one PR once we adopted black.